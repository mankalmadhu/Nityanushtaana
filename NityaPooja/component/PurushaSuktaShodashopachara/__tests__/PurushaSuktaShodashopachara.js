import React from 'react';
import ಪುರುಷಸೂಕ್ತಶೋಢಷೋಪಚಾರ from '../PurushaSuktaShodashopachara';
import Renderer from 'react-test-renderer';

jest.mock('../../Lekhana/Lekhana', () => 'ಲೇಖನ' );
jest.mock('../../Puta/Puta', () => 'ಪುಟ' );
jest.mock('../../Bhandara/Shuklambaradharam/Shuklambaradharam',()=>'ಶುಕ್ಲಾ೦ಬರಧರ೦');

test("ಪುರುಷಸೂಕ್ತಶೋಢಷೋಪಚಾರ", () => {
  expect(Renderer.create(
    <ಪುರುಷಸೂಕ್ತಶೋಢಷೋಪಚಾರ />
  ).toJSON()).toMatchSnapshot();

});