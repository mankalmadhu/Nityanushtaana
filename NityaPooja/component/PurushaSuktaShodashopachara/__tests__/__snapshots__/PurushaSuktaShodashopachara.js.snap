// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಪುರುಷಸೂಕ್ತಶೋಢಷೋಪಚಾರ 1`] = `
<ಲೇಖನ>
  <ಪುಟ>
    <ಶುಕ್ಲಾ೦ಬರಧರ೦ />
  </ಪುಟ>
</ಲೇಖನ>
`;
